cmd/influxd/run.go

```diff
@@ -225,11 +225,17 @@ func openServer(path string, u *url.URL, b *messaging.Broker, initializing, conf
 			openServerClient(s, joinURLs)
 		}
 	} else if !configExists {
-		// We are spining up an server that has no config,
+		// We are spining up a server that has no config,
 		// but already has an initialized data directory
 		joinURLs = []*url.URL{b.URL()}
 		openServerClient(s, joinURLs)
 	} else {
+		if len(joinURLs) == 0 {
+			//This is the first broker, so it always spins up a client to itself for now
```

Comment:
The comment is correct.  It is going to open a client.  Join URLs might be a naming issue that I just get confused by.  It feels like it should be "BrokerURLs", as we only join brokers right?  Even passing in "join" feels too generic to me.  This would be a bigger change though, and I could be wrong.